blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ContainerFilter.java

```diff
@@ -58,7 +59,7 @@
                 }
                 filters[i] = f;
             }
-            Collection<T> out = new ArrayList<>();
+            Collection<T> out = new LinkedList<>();
```

Comment:
Curious .... have we profiled this and identified a performance problem? I assume additions are the main thing, but ??? @vivek ... what's your take?